pom.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.symphony.dal.communicator.axis</groupId>
+    <artifactId>AXIS-M3064-V</artifactId>
+    <version>0.1</version>
+
+	<properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.1</version>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.avispl.symphony.api</groupId>
+            <artifactId>symphony-api-commons</artifactId>
+            <version>5.1.1</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-api-commons-5.1.1.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.avispl.symphony.api</groupId>
+            <artifactId>symphony-api-dal</artifactId>
+            <version>5.1.1</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-api-dal-5.1.1.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.avispl.symphony.dal</groupId>
+            <artifactId>symphony-dal-commons</artifactId>
+            <version>5.1.1</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-dal-commons-5.1.1.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <scope>provided</scope>
+            <version>2.11.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <scope>provided</scope>
+            <version>2.11.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
+            <scope>provided</scope>
+            <version>0.1.55</version>
+        </dependency>
+        <dependency>
+            <groupId>net.java.dev.jna</groupId>
+            <artifactId>jna</artifactId>
+            <scope>provided</scope>
+            <version>5.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>net.java.dev.jna</groupId>
+            <artifactId>jna-platform</artifactId>
+            <version>5.5.0</version>
```

Comment:
please add scope for all the provided dependencies
Please organize dependencies order like provided first, then test dependencies, etc

Reply:
I fixed it